pyuoi/linear_model/base.py

```diff
@@ -453,6 +454,13 @@ def fit(self, X, y, stratify=None, verbose=False):
             self._fit_intercept(X, y)
         self._post_fit(X, y)
 
+        # (Joseph) Write out all parameters to JSON file for later introspection.
```

Comment:
I'm thinking about refactoring this later so that this functionality is wrapped around the `fit` function instead of directly modifying the library code.